prow/cmd/hook/main.go

```diff
@@ -99,6 +102,8 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 
 	fs.StringVar(&o.webhookSecretFile, "hmac-secret-file", "/etc/webhook/hmac", "Path to the file containing the GitHub HMAC secret.")
 	fs.StringVar(&o.slackTokenFile, "slack-token-file", "", "Path to the file containing the Slack token to use.")
+	fs.StringVar(&o.rocketChatWebhookFile, "rocketchat-webhook-file", "", "Path to the file containing the RocketChat webhook url to use.")
```

Comment:
So is the webhook url essentially secret to avoid having to perform hmac or other authz? If so maybe we need to mention it explicitly, endpoint urls are not usually secret :) 

Reading https://docs.rocket.chat/use-rocket.chat/workspace-administration/integrations#incoming-webhook-script it seems that way.

Reply:
You are right, the webhook url is secret and needs no additional authentication.
Is it sufficient to just write `Path to the file containing the secret RocketChat webhook url to use.` or what do you recommend here to mention it explicitly?

Reply:
"Path to the file containing the secret RocketChat webhook url to use." sounds good to me